JS/js-functions_functions-with-parameters-extra/js/index.js

```diff
@@ -19,8 +22,12 @@ printSquare(5);
 
 // Uncomment the following function calls and implement the function printCircumference
 
-// printCircumference(4);
-// printCircumference(6);
+function printCircumference(radius) {
+  const circumference = 2 * 3.14 * radius;
```

Comment:
```suggestion
  const circumference = Math.PI * radius;
```
we can use Math.PI object, if you don't remeber theration :)